RELEASING.md

```diff
@@ -0,0 +1,7 @@
+## Releasing Theme Check Action
+
+1. Check the Semantic Versioning page for info on how to version the new release: http://semver.org
+
+2. Run [`git changelog`](https://github.com/tj/git-extras) to update `CHANGELOG.md`.
+
+3. Follow the [publishing an action](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#publishing-an-action) steps
```

Comment:
Should add steps on rereleasing major and minor tags.

Reply:
`$BIN/rerelease`
```bash
#!/bin/bash

$BIN/git-delete-tag $1
$BIN/release $1
```

`$BIN/git-delete-tag`
```bash
#!/bin/bash

set -eou pipefail

tag=$1
upstream="$(git remote | grep -q upstream && echo 'upstream' || echo 'origin')"
git push --delete $upstream $tag
git tag --delete $tag
```

`$BIN/release`
```zsh
#!/bin/zsh
tag=$1
remote=$({ git remote -v | grep -q upstream } && echo "upstream" || echo "origin")
git tag $1
git push $remote $1
```

Reply:
Thanks for sharing these scripts, @charlespwd! I really appreciate it. I've added them in a new commit :)

Regarding your comment about adding steps for rereleasing major and minor tags, I'm not entirely sure I follow. We do cover the reasoning behind major and minor releases in Step 1, but as far as I can tell, the process for releasing a new version is the same regardless of whether it's a major or minor release. Is that correct?

In any case, I've included extra steps to the document to remove any ambiguity. Let me know if you have any other suggestions.